scripts/test.sh

```diff
@@ -68,17 +68,15 @@ function run_tests {
 trap 'sigint_handler' INT
 
 # build all of Runtime
-./build.sh
+./scripts/build.sh
```

Comment:
This will fail now that this file doesn't exist any more.... does replacing it with `runtime build` work?

Reply:
ah yes. i'll need to replace it with `./runtime build`